airframe-fluentd/src/main/scala/wvlet/airframe/fluentd/FluencyClient.scala

```diff
@@ -14,20 +14,50 @@
 package wvlet.airframe.fluentd
 import java.time.Instant
 
+import org.komamitsu.fluency.fluentd.FluencyBuilderForFluentd
+import org.komamitsu.fluency.ingester.sender.ErrorHandler
 import org.komamitsu.fluency.{EventTime, Fluency}
 import wvlet.log.LogSupport
 import wvlet.airframe._
 
 case class FluencyConfig(
     // Use the extended EventTime timestamps
     // https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1#eventtime-ext-format
-    useExtendedEventTime: Boolean = false,
-    fluencyConfig: Fluency.Config = new Fluency.Config()
+    useExtendedEventTime: java.lang.Boolean = false,
```

Comment:
I think we should use Scala primitive values. In `newFluency` function, we should use Boolean.box(x) to bridge with Java.